### ISSUE TITLE
Impossible to fetch helm chart from local filesystem

### ISSUE THREAD

**Issue by emempi** [ksonnet/ksonnet#848]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
>
/kind feature

**What happened**:
We store some of our helm charts in our private git repository and would like to use it with ksonnet. Usually, we install them e.g. using helm upgrade --install path/to/charts/elasticsearch ... . Although ksonnet is able to support helm chart repositories in GCS, it fails to provide means of using other store -- at least my naive approach has failed:

```
$ ks registry add local ../../charts/
ERROR caching registry: open <absolute path>/charts/registry.yaml: no such file or directory 
```

**What you expected to happen**:
There should be a way to use locally-stored helm charts, either as a repo, or manually importing them one-by-one.

**How to reproduce it (as minimally and precisely as possible)**:
(see "What happened" section")

**Anything else we need to know?**:

**Environment**:
- ksonnet version (use `ks version`):
```
$ ks version
ksonnet version: 0.12.0
jsonnet version: v0.11.2
client-go version: kubernetes-1.10.4
```
- Kubernetes version (use `kubectl version`):
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean", BuildDate:"2018-08-07T23:17:28Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.5-gke.4", GitCommit:"6265b9797fc8680c8395abeab12c1e3bad14069a", GitTreeState:"clean", BuildDate:"2018-08-04T03:47:40Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by bryanl** [comment]
Yes, this correct. It wasn't in our initial feature list, but this is a valid use case. We'll research to see how to proceed from here.
